src/PlotUtils.jl

```diff
@@ -38,6 +38,8 @@ const _default_colorscheme = generate_colorscheme()
 
 if Base.VERSION >= v"1.4.2"
     include("precompile.jl")
+    # optimize_ticks(0., 100.)
```

Comment:
unnecessary comment

Reply:
Do you think we should remove the whole `precompile.j` file since it was the source of allocation issue ?

Reply:
I don't know, I didn't really follow what was going on there.

Reply:
Thanks, I'll merge this and release after an extra maintainer review on the PR.